### PR TITLE
Add missing "I want to help" wiki page

### LINKED ISSUE #80: "I want to help" link is broken
In the 0.49.13 web page notes:

[Please](https://megamek.org/wiki/I-want-to-help) check out this document for contributing to the suite. Deadborder has us covered from land unit sprites and camos. But we desperately need sprites for aerospace units.

But following the link shows the target is gone/has moved:

```
*** Your Page Request DESTROYED by 404 error ***
Sorry, but we couldn't find the page you are looking for.
```

### PR DESCRIPTION
**Pull request by Sleet01**
Fixes #80 

This commit also includes:
1. a Dockerfile that can be used to set up local Jekyll hosting without
   faffing about with Ruby, Jekyll, Bundle, etc.
2. Instructions on how to use the docker container for local hosting
3. Updated links in the recent release posts making links to the missing
   "I want to help" wiki page work both locally and once deployed.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM ruby:2.7-alpine
+WORKDIR /app
+COPY . ./
+RUN apk update
```
Comment (sixlettervariables) [inline Dockerfile:4]: If you move the apk and gem calls before the workdir and copy lines, it'll improve the build times if you modify a file in your local folder (layer reuse).

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+RUN apk add --no-cache build-base gcc cmake git
+RUN gem update --system && gem update bundler && gem install bundler jekyll:3.9.3
+RUN echo "$PWD"
```
Comment (sixlettervariables) [inline Dockerfile:7]: I'd comment this out or remove it as it'll hinder layer reuse.

File: Dockerfile (hunk 1 above)
Comment (Sleet01) [reply Dockerfile:4]: Oh, that's good to know!  I'll make the update.

File: Dockerfile (hunk 2 above)
Comment (Sleet01) [reply Dockerfile:7]: I'll give it a shot.  TY.

Comment (Sleet01) [reply Dockerfile:7]: > I'd comment this out or remove it as it'll hinder layer reuse.

Specifically the `RUN apk add --no-cache build-base gcc cmake git` line?

Comment (Sleet01) [reply Dockerfile:7]: @sixlettervariables Sorry, I'm not sure which line would hinder layer reuse.  It looks like both the `apk` lines and the `gem update` line are required.  I can remove the echo line, though.

Comment (sixlettervariables) [reply Dockerfile:7]: Each line is a new layer, so order matters. If layer 1 changes, all subsequent layers change.

The trick is to order lines based on slowest changing to fastest changing.

The lines you don't need are the echo PWD and the line checking the Jekyll version.

**Comment by Sleet01** [comment]
@sixlettervariables I think that's all the updating I can do.

**Review by rjhancock — APPROVED** [review]

**Pull request merged**